client/src/pages/Home.svelte

```diff
@@ -3,20 +3,58 @@
 
     import { register } from './register.ts';
     import { ButtonType } from '../components/types.ts';
+
+    import Google from '../components/icons/Google.svelte';
+    import Camera from '../components/icons/Camera.svelte';
+    import Search from '../components/icons/Search.svelte';
 </script>
 
 <main>
     {#await register()}
         Waiting for service worker...
     {:then}
-        <div>
-            <a href="/auth/login"><Button type={ButtonType.Primary}>Login with Google</Button></a>
+        <div class="middlecontainer">
+            <img src="../assets/images/logo-background.png" alt="DocTrack Logo" class="image">
+            <br>
+            <h3>DocTrack: Document Tracking System</h3>
+            <a href="/auth/login"><Button type={ButtonType.Primary}><Google />Log in with University of the Philippines Mail</Button></a>
+            <div class="searchcontainer">
+                <input type="text" placeholder="Enter tracking number here" class="trackingfield" />

```

Comment:
Now that the branch has been rebased to main, I think this part should be replaced with the proper `TextInput` component that was merged in #19.
```suggestion
                <TextInput placeholder="Enter tracking number here" class="trackingfield" />

```
Check if the relevant CSS class attached to it is still relevant or if the styles are duplicated by the Svelte component.